flink-python/pyflink/table/table_environment.py

```diff
@@ -719,6 +746,36 @@ def drop_view(self, view_path: str, ignore_if_not_exists: Optional[bool] = True)
         """
         return self._j_tenv.dropView(view_path, ignore_if_not_exists)
 
+    def drop_temporary_model(self, model_path: str) -> bool:
+        """
+        Drops a temporary model registered in the given path.
+
+        If a permanent model with a given path exists, it will be used
+        from now on for any queries that reference this path.
+
+        :param model_path: The path of the registered temporary model.
+        :return: True if a model existed in the given path and was removed.
```

Comment:
nit: a model -> temporary model. 
I assume if there was a permanent model with that path it would still exist.